site/gatsby-site/server/fields/incidents.ts

```diff
@@ -103,6 +103,46 @@ export const mutationFields: GraphQLFieldConfigMap<any, Context> = {
             return updated;
         })
     },
+
+    updateOneIncidentTranslation: {
```

Comment:
Please add a test for this new mutation, should be in /server/tests/incidents.spec.ts